Content.Client/EscapeMenu/UI/Tabs/KeyRebindTab.xaml.cs

```diff
@@ -410,6 +434,7 @@ private sealed class BindButton : Control
             public readonly Button Button;
             public IKeyBinding? Binding;
 
+            [Dependency] private readonly IConfigurationManager configManager = default!;
             public BindButton(KeyRebindTab tab, KeyControl keyControl, string styleClass)
```

Comment:
Pass the configuration manager into the constructor here instead of taking it as `[Dependency]` or doing a `Resolve`.